js/game.js

```diff
@@ -11,23 +11,27 @@ var Question = function (answer, question, message) {
  this.difference = 0;
  this.question = question;
  this.message = message;
+ this.response = ' ';
  this.run = function() {
```

Comment:
You'll also have learned by now that these methods should be moved to prototypes; just wanted to highlight it. It has no real relevance for this assignment.